packages/core/src/notifications/rules.ts

```diff
@@ -96,17 +98,16 @@ export class RuleEngine {
     }
 
     private evaluateRule(criteria: CriteriaCondition) {
-        const expected = criteria.values
+        const expected = criteria.values.map((v) => v.toLowerCase())
```

Comment:
oh wow, is the spec case-insensitive? maybe the spec should be lowercase itself. avoids mistakes in the clients, and reduces variants.

Reply:
No, I just wanted to avoid potentially common mistakes such as `mac` instead of `MAC`. But with proper validation this is not necessary.